client/my-sites/add-ons/hooks/use-add-on-prices.ts

```diff
@@ -0,0 +1,49 @@
+import formatCurrency from '@automattic/format-currency';
+import { useSelector } from 'calypso/state';
+import { getProductCurrencyCode, getProductBySlug } from 'calypso/state/products-list/selectors';
+
+const useAddOnPrices = ( productSlug: string, quantity?: number ) => {
+	return useSelector( ( state ) => {
+		const product = getProductBySlug( state, productSlug );
+		let cost = product?.cost;
+		const currencyCode = getProductCurrencyCode( state, productSlug );
+
+		if ( ! ( cost && currencyCode ) ) {
```

Comment:
Ah that's a keen eye -- the current implementation didn't catch my eye, but I definitely see how it could be simpler to understand since `a || b` is much more common.

I'll make the change 🙂

Reply:
https://github.com/Automattic/wp-calypso/pull/81338/commits/0cd3c9c661bb82373e15bf01b755d53a2c159ff9